tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -117,7 +117,7 @@ do_split_and_parse(unsigned char *str, int type)
 
 	r = split_and_parse_n(str, type, len, chunks);
 	if (++chunks > len)
-		return TFW_STOP;
+		return 1;
```

Comment:
The value now looks strange. Please add a comment that we return any non `TFW_*` constant value just to stop copping a message to more pieces.

Reply:
Corrected.